packages/google_maps_flutter/google_maps_flutter_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.1
```

Comment:
Does it have a public API change, do we need to bump the minor version? Maybe just 0.1.0+2?

Reply:
Yeah, this is not a public api change, I'll try to bump +1... I still don't understand the rules of the validation script to let me use +1s or not!